build.gradle

```diff
@@ -3,7 +3,6 @@ plugins {
     id 'io.github.gradle-nexus.publish-plugin' version '1.1.0' apply false
     id 'com.diffplug.spotless' version '6.1.0'
     id 'net.ltgt.errorprone' version '2.0.2'
-    id 'org.javamodularity.moduleplugin' version '1.8.9'
```

Comment:
:+1: 
Better to stop depending on this plugin, it seems their development is not so active recently. I sent a PR two months ago and there is still no reaction 😂

Reply:
Thanks, that is good to know :)